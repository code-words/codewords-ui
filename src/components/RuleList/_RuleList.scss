.RuleList {
  position: relative;
  top: 10px;
  background: $overlay-filter;
  padding: 15px;
  height: 90%;
  width: 60%;
}

.rules {
  border-top: solid white 2px;
  height: 90%;
  overflow: scroll;

  p {
    margin: 10px;
  }
}