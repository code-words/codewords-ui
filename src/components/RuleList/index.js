import React from 'react';

const RuleList = () => {
  return (
    <article className="RuleList">
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur voluptates earum, est aliquid distinctio, vitae voluptatibus veritatis iusto ab dolore qui odit similique laborum! Quis architecto temporibus nisi dolor nostrum.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur voluptates earum, est aliquid distinctio, vitae voluptatibus veritatis iusto ab dolore qui odit similique laborum! Quis architecto temporibus nisi dolor nostrum.

        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Molestiae nemo ea iure inventore iste laborum saepe, eaque impedit! Molestias quo laborum, voluptatem doloremque magni perspiciatis deleniti enim. Illo, adipisci! Quaerat!
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur voluptates earum, est aliquid distinctio, vitae voluptatibus veritatis iusto ab dolore qui odit similique laborum! Quis architecto temporibus nisi dolor nostrum.
      </p>
    </article>
  )
}

export default RuleList;