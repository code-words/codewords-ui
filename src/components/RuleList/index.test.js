import React from 'react';
import RuleList from '.';
import { shallow } from 'enzyme';

describe('RuleList', () => {
  let wrapper;
  let instance;

  beforeEach(() => {
    wrapper = shallow(<RuleList />);
    instance = wrapper.instance();

  })

  it('should match component snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})