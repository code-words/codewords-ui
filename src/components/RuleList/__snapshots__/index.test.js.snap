// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RuleList should match component snapshot 1`] = `ShallowWrapper {}`;
